static/app/views/organizationGroupDetails/quickTrace/issueQuickTrace.tsx

```diff
@@ -162,6 +170,16 @@ class IssueQuickTrace extends Component<Props, State> {
       <Fragment>
         {this.renderTraceLink(results)}
         <QuickTraceWrapper>
+          {showReplay && (
+            <EventNodeReplay
+              to={`/organizations/${organization.slug}/replays/${project?.slug}:${replayId}`}
```

Comment:
ditto links

Reply:
I made this change but it is worth noting that since the trace isn't always available there's more of a chance we won't have a link to the breadcrumb/replay section even if it's available